.circleci/config.yml

```diff
@@ -15,12 +15,31 @@ jobs:
         key: cli
       - type: shell
         command: |
+          set -e
```

Comment:
this should go as a separate workflow job to run it in parallel. Also, I think this could be cleaned up with the new git tag stuff: https://circleci.com/docs/2.0/workflows/#git-tag-job-execution